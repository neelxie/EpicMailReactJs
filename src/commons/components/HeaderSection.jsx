import React from 'react';
import NavSection from './NavSection';
import '../CSS/Header.scss';

export default function HeaderSection() {
  return (
    <header>
      <NavSection />
    </header>
  );
}
