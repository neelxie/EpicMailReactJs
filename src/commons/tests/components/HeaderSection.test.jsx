/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import HeaderSection from '../../components/HeaderSection';

// test
describe('HeaderSection Component', () => {
  it('should match the snapshot', () => {
    const component = shallow(<HeaderSection />);

    expect(toJson(component)).toMatchSnapshot();
  });
});
