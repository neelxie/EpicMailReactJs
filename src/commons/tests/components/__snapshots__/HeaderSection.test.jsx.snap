// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderSection Component should match the snapshot 1`] = `
<header>
  <NavSection />
</header>
`;
